cves/CVE-2018-8037.yml

```diff
@@ -173,8 +186,10 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
```

Comment:
Was the fix also part of a framework or system behavior related to asynchronous requests? If so, other lessons may apply.